lib/src/config.rs

```diff
@@ -224,24 +224,24 @@ pub struct Curve25519 {
     pub nonce: secretbox::Nonce,
 }
 
-#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
-/// Chunking Algorithms supported by rdedup
-pub enum ChunkingAlogrithm {
-    Bup,
+#[derive(Copy, Clone, Debug, Serialize, Deserialize, PartialEq)]
+/// ```ChunkingAlgorithm``` are the algorithms supported by rdedup
+pub enum ChunkingAlgorithm {
+    /// ```Bup``` is the default algorithm, the u32 value provided with bup is the bit mask size to
+    /// be used by rollsum. The valid range is between 10 and 30 (1KB to 1GB)
+    Bup(u32),
```

Comment:
Yes. I'd like it, so it's self-describing with the key, plus it's possible to add any more keys if it's needed.

Since it's an internal format and is not meant to be edited by humans  `_bits` are OK, especially that bup can only support power of two values. The human readable things are only necessary on the command line, as it's meatbag-facing.